### ISSUE TITLE
Default Layer bounds when unrectified maps are present

### ISSUE THREAD

**Issue by lklepner** [nypl-spacetime/nypl-warper#31]
For some reason the Show tab of Layer 1112 is defaulting to bounds which are extremely close and seemingly random - http://maps.nypl.org/warper/layers/1112

This layer contains a number of unrectified maps. Is it possible that the existence of unrectified maps is causing the bounds calculation to fail?

It seems like the ideal behavior in this case would be to default the bounds to the rectified maps in the layer.

**Comment by mgiraldo** [comment]
yeah there's a lot of loose ends in many maps/layers... we're working on a way to find out those outliers and fix them
